core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpAdapter.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.help
+
+import android.animation.ObjectAnimator
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.item_help.item_help_description
+import kotlinx.android.synthetic.main.item_help.item_help_title
+import kotlinx.android.synthetic.main.item_help.item_help_toggle_expand
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.utils.AnimationUtils.collapse
+import org.kiwix.kiwixmobile.core.utils.AnimationUtils.expand
+import org.kiwix.kiwixmobile.core.base.adapter.BaseViewHolder
+
+internal class HelpAdapter(titleDescriptionMap: Map<String, String>) :
+  RecyclerView.Adapter<HelpAdapter.Item>() {
+  private var helpItems: ArrayList<HelpItem> = ArrayList()
+
+  init {
+    titleDescriptionMap.forEach { (key, value) ->
+      helpItems.add(HelpItem(key, value))
+    }
+  }
+
+  override fun onCreateViewHolder(
+    parent: ViewGroup,
+    viewType: Int
+  ): Item = Item(
+    LayoutInflater.from(parent.context)
```

Comment:
we have an extension function for inflation on `Context`

Reply:
@macgills I Can't find it.